src/app/src/reducers/UIReducer.js

```diff
@@ -20,6 +21,10 @@ const initialState = Object.freeze({
         filterText: '',
         resetButtonClickCount: 0,
     }),
+    window: Object.freeze({
+        innerHeight: typeof window === 'object' ? window.innerHeight : null,
+        innerWidth: typeof window === 'object' ? window.innerWidth : null,
```

Comment:
There is also https://lodash.com/docs/4.17.15#isObject but it probably does the same thing.

Reply:
Interestingly there is a subtle difference. `_.isObject` considers a function to be an "object"

https://github.com/lodash/lodash/blob/9f865886f8ae22b205107b00b7b65871bfc9a2f0/lodash.isobject/index.js#L35-L38

Good to know, but it doesn't matter in this case. Switching to `isObject` will make things a bit more readable.

Reply:
Switched to using `isObject` in fixup 2952bdf